src/app/components/audio-recordings/components/download-table/download-table.component.html

```diff
@@ -0,0 +1,53 @@
+<ngx-datatable
+  bawDatatableDefaults
+  bawDatatablePagination
+  [filters$]="filters$"
+  [models]="getModels"
```

Comment:
probably worth prefixing these inputs so readers know which directive they belong to.

`pagination-filter$` and `pagination-getter`?